docs/Developers/Debug_Bears.rst

```diff
@@ -122,3 +122,88 @@ Or to debug all bears specified by ``bears`` setting:
     bears = PEP8Bear,MypyBear
     files = <filename>
     debug_bears = True
+
+Debugger features
+-----------------
+
+Inspect Bear settings
+^^^^^^^^^^^^^^^^^^^^^
+
+New command ``settings`` included in coala's Debugging Interface to inspect
+Bear settings in debugging environment, ``settings`` command displays the
+updated setting of a Bear i.e., user can quickly inspect the passed settings.
```

Comment:
-->
**A** new command ``settings`` **is** included in [...] Bear settings in **the** debugging environment. It displays all passed settings of a Bear and their values so you can quickly inspect them.